django/thunderstore/frontend/templates/base.html

```diff
@@ -170,11 +182,12 @@ <h2>Developers</h2>
                         <a href="{% url 'tools.manifest-v1-validator' %}">Manifest Validator</a>
                     </div>
                 </div>
+                {% endcache %}
             </div>
             <div class="container footer_links">
                 <p>Thunderstore 2023</p>
                 <div>
-                    <a class="nav-link" href="https://discord.gg/UWpWhjZken"><span class="fab fa-discord"></span></a>
+                    <a class="nav-link" href="https://discord.gg/UWpWhjZken"><span class="fab fa-discord" style="min-height: 24px"></span></a>
```

Comment:
Hacky.

Reply:
But simpler than having to look up a custom class declared just for this single use case